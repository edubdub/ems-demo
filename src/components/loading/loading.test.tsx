import * as React from 'react'
import { shallow } from 'enzyme'
import Loading from './loading'

it('renders without crashing', () => {
  expect(shallow(<Loading />)).toMatchSnapshot()
})
