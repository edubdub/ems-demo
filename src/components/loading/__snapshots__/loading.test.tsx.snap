// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <img
        className="loadingSpinner"
        src="ems-software.png"
/>,
      "circle": true,
      "rounded": true,
      "style": Object {
        "display": "block",
        "left": "50%",
        "position": "fixed",
        "textAlign": "center",
        "top": 100,
        "transform": "translateX(-50%)",
      },
      "transitionEnabled": true,
      "zDepth": 1,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "className": "loadingSpinner",
        "src": "ems-software.png",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <img
          className="loadingSpinner"
          src="ems-software.png"
/>,
        "circle": true,
        "rounded": true,
        "style": Object {
          "display": "block",
          "left": "50%",
          "position": "fixed",
          "textAlign": "center",
          "top": 100,
          "transform": "translateX(-50%)",
        },
        "transitionEnabled": true,
        "zDepth": 1,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "className": "loadingSpinner",
          "src": "ems-software.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
